stdlib/xml/dom/minidom.pyi

```diff
@@ -52,8 +52,6 @@ class DocumentFragment(Node):
     nodeName: str
     nodeValue: Incomplete
     attributes: Incomplete
-    parentNode: Incomplete
-    childNodes: Incomplete
```

Comment:
Can we maybe also leave the ones that are explicitly marked as `Incomplete` as they are? Since we've explicitly marked these as "we haven't figured out what they're meant to be yet", they might plausibly have different types in the subclass than in the superclass at runtime.

Reply:
I want to cancel all changes in this file :)
Because after my merge conflict I stopped understanding what is going on.